site/zenodo_rdm/migrator/transform/records.py

```diff
@@ -10,17 +10,17 @@
 from invenio_rdm_migrator.streams.records import RDMRecordTransform
 
 from .entries.records.record_files import ZenodoRDMRecordFileEntry
-from .entries.records.records import ZenodoRecordEntry
+from .entries.records.records import ZenodoDraftEntry, ZenodoRecordEntry
 
 
 class ZenodoRecordTransform(RDMRecordTransform):
     """Zenodo to RDM Record class for data transformation."""
 
     def _communities(self, entry):
-        communities = entry["json"].get("communities")
-        if communities:
-            slugs = [slug for slug in communities]
-            return {"ids": slugs, "default": slugs[0]}
+        # communities = entry["json"].get("communities")
```

Comment:
can we put this back or there is something wrong with drafts that belong already to communities?